.storybook/main.ts

```diff
@@ -13,6 +16,18 @@ const config: StorybookConfig = {
     name: '@storybook/react-vite',
     options: {},
   },
+  viteFinal: async (config) => {
+    return mergeConfig(config, {
+      resolve: {
+        alias: {
+          '@/core': path.resolve(__dirname, '../src/core'),
+          '@/core-react': path.resolve(__dirname, '../src/core-react'),
+          '@/ui-react': path.resolve(__dirname, '../src/ui/react'),
+          '@': path.resolve(__dirname, '../src'),
```

Comment:
needs to be `@/`?

Reply:
nope, just @, matching https://github.com/coinbase/onchainkit/blob/main/vitest.config.ts#L40